XSIAM | Bluecat Address Manager ModelingRules (#26368)
demisto/content: Packs/BluecatAddressManager/README.md

# Bluecat Address Manager
This pack includes Cortex XSIAM content. 

## Configuration on Server Side
You need to configure Bluecat Address Manager to forward Syslog messages to XSIAM.

Please proceed with the following steps to configure syslog redirection on the BAM server:
1. Select the **Administration** tab.
2. Under **General**, click **Service Configuration**.

> 3. From the **Service Type** drop-down menu, select **Syslog**.

Reviewer: ```suggestion
3. From the **Service Type** dropdown menu, select **Syslog**.
```

> 4. Under **General Settings**, set the following parameters: Under Syslog Server — enter the IPv4 address of the BrokerVM and click **Add**. 

Reviewer: ```suggestion
4. Under **General Settings** > **Syslog Server** enter the IPv4 address of the BrokerVM and click **Add**. 
```

# Bluecat Address Manager
This pack includes Cortex XSIAM content. 

## Configuration on Server Side
You need to configure Bluecat Address Manager to forward Syslog messages to XSIAM in UTC format.

#### Please proceed with the following steps to configure syslog redirection on the BAM server:
1. Select the **Administration** tab.
2. Under **General**, click **Service Configuration**.
3. From the **Service Type** dropdown menu, select **Syslog**.
4. Under **General Settings** > **Syslog Server** enter the IPv4 address of the BrokerVM and click **Add**. 
5. Click **Update**.

> #### To configure the BAM server to send logs in UTC format, please proceed with the following steps:

Reviewer: ```suggestion
#### To configure the BAM server to send logs in UTC format, complete the following steps:
```

> #### To configure the BAM server to send logs in UTC format, please proceed with the following steps:

Reviewer: ```suggestion
#### To configure the BAM server to send logs in UTC format, complete the following steps:
```

1. Log in to the BlueCat Address Manager web interface using your administrator credentials.

> 2. Click on the **Administration** tab located at the top of the page.

Reviewer: ```suggestion
2. Click the **Administration** tab located at the top of the page.
```

> 3. In the left-hand navigation pane, click on **System** and then select **System Configuration**.

Reviewer: ```suggestion
3. In the left-hand navigation pane, click **System** and then select **System Configuration**.
```

4. In the **System Configuration** page, you will see various configuration options. Look for the **Timezone** section.

> 5. Click on the **Edit** button next to the timezone field.

Reviewer: ```suggestion
5. Click the **Edit** button next to the **Timezone** field.
```

6. A pop-up window or a dropdown menu will appear, depending on the version of BlueCat Address Manager you are using.
  - If it's a pop-up window, select the desired timezone from the list provided and click **Save** or **Apply** to save the changes.

>   -  If it's a dropdown menu, click on the dropdown arrow and select the desired timezone from the list. Then click on **Save** or **Apply** to save the changes.

Reviewer: ```suggestion
  -  If it's a dropdown menu, click the dropdown arrow and select the desired timezone from the list. Then click **Save** or **Apply** to save the changes.
```
